qt5ct fails to compile on master

```
[130] milan@nixos> nix-shell -p qt5ct                                                                                /etc/nixos/nixpkgs
warning: Nix search path entry '/etc/nixos/overlay' does not exist, ignoring
these derivations will be built:
  /nix/store/s6da99asaf232c8ijcp43g8wvq6z3yg3-qt5ct-0.41.drv
building '/nix/store/s6da99asaf232c8ijcp43g8wvq6z3yg3-qt5ct-0.41.drv'...
qmakePrePhase
unpacking sources
unpacking source archive /nix/store/b7pmjsqckf0rq7dr1gk0dqm9ba5vnzbw-qt5ct-0.41.tar.bz2
source root is qt5ct-0.41
setting SOURCE_DATE_EPOCH to timestamp 1567626766 of file qt5ct-0.41/qt5ct.pri
patching sources
configuring
QMAKEPATH=/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev:/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev:/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev:/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev:/nix/store/i6w27ri0dkydjdbpvhi9l8sbhqksw3fq-qtwayland-5.12.4-dev:/nix/store/i6w27ri0dkydjdbpvhi9l8sbhqksw3fq-qtwayland-5.12.4-dev:/nix/store/cw4y4z9l2phyj0ffz7lrjc0k7hl8kvpv-qtdeclarative-5.12.4-dev:/nix/store/cw4y4z9l2phyj0ffz7lrjc0k7hl8kvpv-qtdeclarative-5.12.4-dev:/nix/store/z1mmnbw99qshq411fz15z617mdfmmmi7-qtsvg-5.12.4-dev:/nix/store/z1mmnbw99qshq411fz15z617mdfmmmi7-qtsvg-5.12.4-dev
qmake PREFIX=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins NIX_OUTPUT_OUT=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DEV=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_BIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DOC=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/share/doc/qt-5.12.4 NIX_OUTPUT_QML=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/qml NIX_OUTPUT_PLUGIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins CONFIG+=release LRELEASE_EXECUTABLE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease QMAKE_LRELEASE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease
Info: creating stash file /build/qt5ct-0.41/.qmake.stash
Project MESSAGE: Found lrelease executable: /nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease
Project MESSAGE: generating translations
Updating './src/qt5ct/translations/qt5ct_zh_CN.qm'...
    Generated 119 translation(s) (117 finished and 2 unfinished)
    Ignored 24 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_en.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_he.qm'...
    Generated 113 translation(s) (111 finished and 2 unfinished)
    Ignored 30 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_da.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_lt.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_zh_TW.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_bg.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_it.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_fi.qm'...
    Generated 135 translation(s) (135 finished and 0 unfinished)
    Ignored 8 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_tr.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_ca.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_gl_ES.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_sr_BA.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_nl.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_uk_UA.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_kk.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_cs.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_ja.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_nl_NL.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_sr_RS.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_uk.qm'...
    Generated 15 translation(s) (14 finished and 1 unfinished)
    Ignored 128 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_el.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_pt_BR.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_pl.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_de.qm'...
    Generated 142 translation(s) (142 finished and 0 unfinished)
    Ignored 1 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_sk.qm'...
    Generated 120 translation(s) (118 finished and 2 unfinished)
    Ignored 23 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_hu.qm'...
    Generated 75 translation(s) (71 finished and 4 unfinished)
    Ignored 68 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_fr.qm'...
    Generated 120 translation(s) (118 finished and 2 unfinished)
    Ignored 23 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_it_IT.qm'...
    Generated 128 translation(s) (126 finished and 2 unfinished)
    Ignored 15 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_ar.qm'...
    Generated 127 translation(s) (125 finished and 2 unfinished)
    Ignored 16 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_pt.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
    Ignored 143 untranslated source text(s)
Updating './src/qt5ct/translations/qt5ct_ru.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_es_ES.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_es.qm'...
    Generated 143 translation(s) (143 finished and 0 unfinished)
Updating './src/qt5ct/translations/qt5ct_sr.qm'...
    Generated 128 translation(s) (126 finished and 2 unfinished)
    Ignored 15 untranslated source text(s)
Project MESSAGE: PREFIX=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins
Project MESSAGE: BINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/bin
Project MESSAGE: DATADIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share
Project MESSAGE: PLUGINDIR=/nix/store/82sgl40m0k3b1qbkdrny8p9n5lyaq8xz-qtbase-5.12.4/lib/qt-5.12.4/plugins
building
build flags: -j4 -l4 SHELL=/nix/store/v2x1ks46fq3yn8c94qfv38wrvmr5xn5r-bash-4.4-p23/bin/bash
cd src/qt5ct-style/ && ( test -e Makefile || /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/qmake -o Makefile /build/qt5ct-0.41/src/qt5ct-style/qt5ct-style.pro 'PREFIX=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins' NIX_OUTPUT_OUT=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DEV=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_BIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DOC=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/share/doc/qt-5.12.4 NIX_OUTPUT_QML=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/qml NIX_OUTPUT_PLUGIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins CONFIG+=release LRELEASE_EXECUTABLE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease QMAKE_LRELEASE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease ) && make -f Makefile
cd src/qt5ct-qtplugin/ && ( test -e Makefile || /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/qmake -o Makefile /build/qt5ct-0.41/src/qt5ct-qtplugin/qt5ct-qtplugin.pro 'PREFIX=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins' NIX_OUTPUT_OUT=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DEV=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_BIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DOC=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/share/doc/qt-5.12.4 NIX_OUTPUT_QML=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/qml NIX_OUTPUT_PLUGIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins CONFIG+=release LRELEASE_EXECUTABLE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease QMAKE_LRELEASE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease ) && make -f Makefile
cd src/qt5ct/ && ( test -e Makefile || /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/qmake -o Makefile /build/qt5ct-0.41/src/qt5ct/qt5ct.pro 'PREFIX=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 PLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins' NIX_OUTPUT_OUT=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DEV=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_BIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 NIX_OUTPUT_DOC=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/share/doc/qt-5.12.4 NIX_OUTPUT_QML=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/qml NIX_OUTPUT_PLUGIN=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins CONFIG+=release LRELEASE_EXECUTABLE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease QMAKE_LRELEASE=/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/bin/lrelease ) && make -f Makefile
Project MESSAGE: D-Bus support: Enabled
Project MESSAGE: This project is using private headers and will therefore be tied to this specific Qt module build version.
Project MESSAGE: Running this project against other versions of the Qt modules may crash at any arbitrary point.
Project MESSAGE: This is not a bug, but a result of using Qt internals. You have been warned!
Project MESSAGE: This project is using private headers and will therefore be tied to this specific Qt module build version.
Project MESSAGE: Running this project against other versions of the Qt modules may crash at any arbitrary point.
Project MESSAGE: This is not a bug, but a result of using Qt internals. You have been warned!
make[1]: Entering directory '/build/qt5ct-0.41/src/qt5ct-style'
g++ -pipe -O2 -std=gnu++11 -Wall -W -dM -E -o .build/moc/moc_predefs.h /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/features/data/dummy.cpp
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -I.. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/qt5ctproxystyle.o qt5ctproxystyle.cpp
make[1]: Entering directory '/build/qt5ct-0.41/src/qt5ct-qtplugin'
g++ -pipe -O2 -std=gnu++11 -Wall -W -dM -E -o .build/moc/moc_predefs.h /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/features/data/dummy.cpp
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -I.. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/qt5ct.o ../qt5ct/qt5ct.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_THEME_SUPPORT_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_DBUS_LIB -DQT_CORE_LIB -I. -I.. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4/QtThemeSupport -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtDBus -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/qt5ctplatformtheme.o qt5ctplatformtheme.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
make[1]: Entering directory '/build/qt5ct-0.41/src/qt5ct'
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic mainwindow.ui -o .build/ui/ui_mainwindow.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic appearancepage.ui -o .build/ui/ui_appearancepage.h
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic fontspage.ui -o .build/ui/ui_fontspage.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic paletteeditdialog.ui -o .build/ui/ui_paletteeditdialog.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic iconthemepage.ui -o .build/ui/ui_iconthemepage.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic interfacepage.ui -o .build/ui/ui_interfacepage.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic fontconfigdialog.ui -o .build/ui/ui_fontconfigdialog.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic previewform.ui -o .build/ui/ui_previewform.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic qsspage.ui -o .build/ui/ui_qsspage.h
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/uic qsseditordialog.ui -o .build/ui/ui_qsseditordialog.h
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/main.o main.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
<command-line>: error: missing terminating " character
<command-line>: note: in definition of macro 'QT5CT_DATADIR'
<command-line>: error: missing terminating " character
<command-line>: note: in definition of macro 'QT5CT_DATADIR'
make[1]: *** [Makefile:468: .build/obj/qt5ct.o] Error 1
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx7make[1]: *** Waiting for unfinished jobs....
0ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/mainwindow.o mainwindow.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
make[1]: Leaving directory '/build/qt5ct-0.41/src/qt5ct-style'
make: *** [Makefile:74: sub-src-qt5ct-style-make_first] Error 2
make: *** Waiting for unfinished jobs....
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/tabpage.o tabpage.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/appearancepage.o appearancepage.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/fontspage.o fontspage.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_THEME_SUPPORT_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_DBUS_LIB -DQT_CORE_LIB -I. -I.. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4/QtThemeSupport -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtDBus -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/qt5ct.o ../qt5ct/qt5ct.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/moc -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_THEME_SUPPORT_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_DBUS_LIB -DQT_CORE_LIB --include /build/qt5ct-0.41/src/qt5ct-qtplugin/.build/moc/moc_predefs.h -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -I/build/qt5ct-0.41/src/qt5ct-qtplugin -I/build/qt5ct-0.41/src -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4 -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4/QtThemeSupport -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtDBus -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include -I/nix/store/w6q39yxxy2zb5fpn85n1q256l3r0g94w-libxml2-2.9.9-dev/include -I/nix/store/m9ad1h95v1bp687kinx7yq8w1fkr9ia6-zlib-1.2.11-dev/include -I/nix/store/a6p1xk0zin35751ikzqwsysfbz4afp3z-libxslt-1.1.33-dev/include -I/nix/store/s0p7kv6syvwh3nnlvvbd3y1ydq21zr61-openssl-1.1.1d-dev/include -I/nix/store/pw8lyyp69y7nmznww31jn4lllhcg1h3a-sqlite-3.28.0-dev/include -I/nix/store/7knsy7ra4ax7f929k2hfqw6dy07pqarj-harfbuzz-2.6.1-dev/include -I/nix/store/hgr0hi5m3z1wl4vwlvvqyl263ddslm86-graphite2-1.3.6/include -I/nix/store/hdy4lm2smp9lmkvjf9aycag0vpskpzqm-icu4c-64.2-dev/include -I/nix/store/zah8q96amg2s53rkhy2dwm239wysvznz-libjpeg-turbo-2.0.3-dev/include -I/nix/store/chw4gj3ryxm8arfx3k4inqld9zmlqb19-libpng-apng-1.6.37-dev/include -I/nix/store/1a6rwzrf7a5av74ryk3pyw7mka83ki7j-libtiff-4.0.10-dev/include -I/nix/store/3gdcszhc8x1h6hyihxrlc31ppj84nzz8-xz-5.2.4-dev/include -I/nix/store/8yrzqd88fk8d65ssn8xs3i4s1s5ddxkw-pcre2-10.33-dev/include -I/nix/store/sna4dvx8p74b1q2nawa3jzmrwd8s36pj-dbus-1.12.16-dev/include -I/nix/store/3h2g61icqv4smw54hdjcipaickffv7cp-expat-2.2.8-dev/include -I/nix/store/vn8cxw1ybn1nhakmsn46r7pgavvmgkf9-glib-2.62.0-dev/include -I/nix/store/zm6n9adsxjncqrqsmdas4q8wcf9adhhl-libffi-3.2.1-dev/include -I/nix/store/amfhrrzba1x3igvfaqwyzgv6qjr2qjjp-gettext-0.19.8.1/include -I/nix/store/3fkh0k2ymbg59z0m4gsvx7g76qn6gib8-glibc-iconv-2.27/include -I/nix/store/d4606zkyn6ajx1f05cngbia3mfnkgqzf-systemd-243-dev/include -I/nix/store/m6s2ljnpslml87z2ivbcajsa4kmc07i5-fontconfig-2.12.6-dev/include -I/nix/store/yk7nf2h1xgb7nbmlc8jjyz86668f3gsa-freetype-2.10.1-dev/include -I/nix/store/7688s3f435gqjrzc51cibxcc92rrmgjk-bzip2-1.0.6.0.1-dev/include -I/nix/store/cfav9l7wky9sxqx3xbn0g0c1rfy1lr49-libX11-1.6.8-dev/include -I/nix/store/b0ns9l6cmzc7dknj34689xl9wh068v2g-xorgproto-2018.4/include -I/nix/store/njl9k0x4l83fbd6ysivjaa87grgsdj8v-libxcb-1.13.1-dev/include -I/nix/store/avqfqvfs3pcw9pc6g319cyajirnmr2im-libXcomposite-0.4.5-dev/include -I/nix/store/69zkjr381yw2h274p2j8kjwadzw3460p-libXfixes-5.0.3-dev/include -I/nix/store/la8lmav29wfj02408qgdwcdj1df73ycz-libXext-1.3.4-dev/include -I/nix/store/z40wsvhvqdzi776slkxpnr2i5xb0l5nq-libXau-1.0.9-dev/include -I/nix/store/3rzgq4ajq5sshjndm1sg3djgc17aajww-libXi-1.7.10-dev/include -I/nix/store/b3zak9z3ryydn3dfcj5knmyrf8bmyjc1-libXrender-0.9.10-dev/include -I/nix/store/qdfxxn2rslwih6a244d1idq4g519xwyk-libxkbcommon-0.8.4-dev/include -I/nix/store/5gh86s1942v7si5k3hlwvy3ch9jgrh91-xcb-util-0.4.0-dev/include -I/nix/store/x9xj6013xml25g5vg26qr2bkkkwykw8k-xcb-util-image-0.4.0-dev/include -I/nix/store/0d2l402w0m6v298s6hvpacjfsqiyb8gr-xcb-util-keysyms-0.4.0-dev/include -I/nix/store/r23nrml1jkhq2db4yjhixl3fm58c0n9g-xcb-util-renderutil-0.3.9-dev/include -I/nix/store/gqqb7nh01c7jpilxpvx6qb2mqibldmqb-xcb-util-wm-0.4.1-dev/include -I/nix/store/4yd4vq7ipnixg52qhi3w5mzdd18frm32-libGL-1.0.0-dev/include -I/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/include -I/nix/store/i6w27ri0dkydjdbpvhi9l8sbhqksw3fq-qtwayland-5.12.4-dev/include -I/nix/store/cw4y4z9l2phyj0ffz7lrjc0k7hl8kvpv-qtdeclarative-5.12.4-dev/include -I/nix/store/z1mmnbw99qshq411fz15z617mdfmmmi7-qtsvg-5.12.4-dev/include -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include/c++/8.3.0 -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include/c++/8.3.0/x86_64-unknown-linux-gnu -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include/c++/8.3.0/backward -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/lib/gcc/x86_64-unknown-linux-gnu/8.3.0/include -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/lib/gcc/x86_64-unknown-linux-gnu/8.3.0/include-fixed -I/nix/store/m48mayi9siabcix556g1pcccsy3g80kn-glibc-2.27-dev/include qt5ctplatformtheme.h -o .build/moc/moc_qt5ctplatformtheme.cpp
<command-line>: error: missing terminating " character
<command-line>: note: in definition of macro 'QT5CT_DATADIR'
<command-line>: error: missing terminating " character
<command-line>: note: in definition of macro 'QT5CT_DATADIR'
/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/bin/moc -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_THEME_SUPPORT_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_DBUS_LIB -DQT_CORE_LIB --include /build/qt5ct-0.41/src/qt5ct-qtplugin/.build/moc/moc_predefs.h -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -I/build/qt5ct-0.41/src/qt5ct-qtplugin -I/build/qt5ct-0.41/src -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4 -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4/QtThemeSupport -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtDBus -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include -I/nix/store/w6q39yxxy2zb5fpn85n1q256l3r0g94w-libxml2-2.9.9-dev/include -I/nix/store/m9ad1h95v1bp687kinx7yq8w1fkr9ia6-zlib-1.2.11-dev/include -I/nix/store/a6p1xk0zin35751ikzqwsysfbz4afp3z-libxslt-1.1.33-dev/include -I/nix/store/s0p7kv6syvwh3nnlvvbd3y1ydq21zr61-openssl-1.1.1d-dev/include -I/nix/store/pw8lyyp69y7nmznww31jn4lllhcg1h3a-sqlite-3.28.0-dev/include -I/nix/store/7knsy7ra4ax7f929k2hfqw6dy07pqarj-harfbuzz-2.6.1-dev/include -I/nix/store/hgr0hi5m3z1wl4vwlvvqyl263ddslm86-graphite2-1.3.6/include -I/nix/store/hdy4lm2smp9lmkvjf9aycag0vpskpzqm-icu4c-64.2-dev/include -I/nix/store/zah8q96amg2s53rkhy2dwm239wysvznz-libjpeg-turbo-2.0.3-dev/include -I/nix/store/chw4gj3ryxm8arfx3k4inqld9zmlqb19-libpng-apng-1.6.37-dev/include -I/nix/store/1a6rwzrf7a5av74ryk3pyw7mka83ki7j-libtiff-4.0.10-dev/include -I/nix/store/3gdcszhc8x1h6hyihxrlc31ppj84nzz8-xz-5.2.4-dev/include -I/nix/store/8yrzqd88fk8d65ssn8xs3i4s1s5ddxkw-pcre2-10.33-dev/include -I/nix/store/sna4dvx8p74b1q2nawa3jzmrwd8s36pj-dbus-1.12.16-dev/include -I/nix/store/3h2g61icqv4smw54hdjcipaickffv7cp-expat-2.2.8-dev/include -I/nix/store/vn8cxw1ybn1nhakmsn46r7pgavvmgkf9-glib-2.62.0-dev/include -I/nix/store/zm6n9adsxjncqrqsmdas4q8wcf9adhhl-libffi-3.2.1-dev/include -I/nix/store/amfhrrzba1x3igvfaqwyzgv6qjr2qjjp-gettext-0.19.8.1/include -I/nix/store/3fkh0k2ymbg59z0m4gsvx7g76qn6gib8-glibc-iconv-2.27/include -I/nix/store/d4606zkyn6ajx1f05cngbia3mfnkgqzf-systemd-243-dev/include -I/nix/store/m6s2ljnpslml87z2ivbcajsa4kmc07i5-fontconfig-2.12.6-dev/include -I/nix/store/yk7nf2h1xgb7nbmlc8jjyz86668f3gsa-freetype-2.10.1-dev/include -I/nix/store/7688s3f435gqjrzc51cibxcc92rrmgjk-bzip2-1.0.6.0.1-dev/include -I/nix/store/cfav9l7wky9sxqx3xbn0g0c1rfy1lr49-libX11-1.6.8-dev/include -I/nix/store/b0ns9l6cmzc7dknj34689xl9wh068v2g-xorgproto-2018.4/include -I/nix/store/njl9k0x4l83fbd6ysivjaa87grgsdj8v-libxcb-1.13.1-dev/include -I/nix/store/avqfqvfs3pcw9pc6g319cyajirnmr2im-libXcomposite-0.4.5-dev/include -I/nix/store/69zkjr381yw2h274p2j8kjwadzw3460p-libXfixes-5.0.3-dev/include -I/nix/store/la8lmav29wfj02408qgdwcdj1df73ycz-libXext-1.3.4-dev/include -I/nix/store/z40wsvhvqdzi776slkxpnr2i5xb0l5nq-libXau-1.0.9-dev/include -I/nix/store/3rzgq4ajq5sshjndm1sg3djgc17aajww-libXi-1.7.10-dev/include -I/nix/store/b3zak9z3ryydn3dfcj5knmyrf8bmyjc1-libXrender-0.9.10-dev/include -I/nix/store/qdfxxn2rslwih6a244d1idq4g519xwyk-libxkbcommon-0.8.4-dev/include -I/nix/store/5gh86s1942v7si5k3hlwvy3ch9jgrh91-xcb-util-0.4.0-dev/include -I/nix/store/x9xj6013xml25g5vg26qr2bkkkwykw8k-xcb-util-image-0.4.0-dev/include -I/nix/store/0d2l402w0m6v298s6hvpacjfsqiyb8gr-xcb-util-keysyms-0.4.0-dev/include -I/nix/store/r23nrml1jkhq2db4yjhixl3fm58c0n9g-xcb-util-renderutil-0.3.9-dev/include -I/nix/store/gqqb7nh01c7jpilxpvx6qb2mqibldmqb-xcb-util-wm-0.4.1-dev/include -I/nix/store/4yd4vq7ipnixg52qhi3w5mzdd18frm32-libGL-1.0.0-dev/include -I/nix/store/zkpvl4v1d7sh83w9dkvp1smydqhhsm4p-qttools-5.12.4-dev/include -I/nix/store/i6w27ri0dkydjdbpvhi9l8sbhqksw3fq-qtwayland-5.12.4-dev/include -I/nix/store/cw4y4z9l2phyj0ffz7lrjc0k7hl8kvpv-qtdeclarative-5.12.4-dev/include -I/nix/store/z1mmnbw99qshq411fz15z617mdfmmmi7-qtsvg-5.12.4-dev/include -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include/c++/8.3.0 -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include/c++/8.3.0/x86_64-unknown-linux-gnu -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include/c++/8.3.0/backward -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/lib/gcc/x86_64-unknown-linux-gnu/8.3.0/include -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/include -I/nix/store/bhngps8y3sf2hdfkbi16bk2ya3k67rkq-gcc-8.3.0/lib/gcc/x86_64-unknown-linux-gnu/8.3.0/include-fixed -I/nix/store/m48mayi9siabcix556g1pcccsy3g80kn-glibc-2.27-dev/include main.cpp -o .build/moc/main.moc
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DQT_NO_DEBUG -DQT_PLUGIN -DQT_THEME_SUPPORT_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_DBUS_LIB -DQT_CORE_LIB -I. -I.. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtThemeSupport/5.12.4/QtThemeSupport -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtDBus -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/moc_qt5ctplatformtheme.o .build/moc/moc_qt5ctplatformtheme.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/qt5ct.o qt5ct.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
make[1]: *** [Makefile:477: .build/obj/qt5ct.o] Error 1
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx7make[1]: *** Waiting for unfinished jobs....
0ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/paletteeditdialog.o paletteeditdialog.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
<command-line>: error: missing terminating " character
<command-line>: note: in definition of macro 'QT5CT_DATADIR'
<command-line>: error: missing terminating " character
<command-line>: note: in definition of macro 'QT5CT_DATADIR'
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/iconthemepage.o iconthemepage.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
make[1]: Leaving directory '/build/qt5ct-0.41/src/qt5ct-qtplugin'
make: *** [Makefile:49: sub-src-qt5ct-qtplugin-make_first] Error 2
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT5CT_DATADIR=\"/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41 -DPLUGINDIR=/nix/store/s617f1krpkshg875204vqvglf1xg52lz-qt5ct-0.41/lib/qt-5.12.4/plugins/share\" -DUSE_WIDGETS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CONCURRENT_LIB -DQT_CORE_LIB -I. -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtWidgets -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui/5.12.4/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtGui -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtConcurrent -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4 -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore/5.12.4/QtCore -isystem /nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/include/QtCore -I.build/moc -I/nix/store/8vq1m2zwxizhvwg9ndlbgac36m21ficg-mesa-19.1.5-dev/include -I.build/ui -I/nix/store/i3syx70ksx800gr29w5kvya1vqhzlirx-qtbase-5.12.4-dev/mkspecs/linux-g++ -o .build/obj/interfacepage.o interfacepage.cpp
<command-line>: warning: missing terminating " character
<command-line>: warning: missing terminating " character
make[1]: *** [Makefile:670: .build/obj/qt5ct.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/build/qt5ct-0.41/src/qt5ct'
make: *** [Makefile:99: sub-src-qt5ct-make_first] Error 2
builder for '/nix/store/s6da99asaf232c8ijcp43g8wvq6z3yg3-qt5ct-0.41.drv' failed with exit code 2
error: build of '/nix/store/s6da99asaf232c8ijcp43g8wvq6z3yg3-qt5ct-0.41.drv' failed
[100] milan@nixos> 
```

--- a/pkgs/tools/misc/qt5ct/default.nix
+++ b/pkgs/tools/misc/qt5ct/default.nix
@@ -1,4 +1,4 @@
-{ mkDerivation, lib, fetchurl, qtbase, qttools, qmake }:
+{ mkDerivation, lib, fetchurl, qtbase, qtsvg, qttools, qmake }:
 
 let inherit (lib) getDev; in
 
@@ -13,15 +13,12 @@ mkDerivation rec {
 
   nativeBuildInputs = [ qmake qttools ];
 
-  buildInputs = [ qtbase ];
+  buildInputs = [ qtbase qtsvg ];
 
   qmakeFlags = [
     "LRELEASE_EXECUTABLE=${getDev qttools}/bin/lrelease"
+    "PLUGINDIR=${placeholder "out"}/${qtbase.qtPluginPrefix}"
   ];
-
-  preConfigure = ''
-    qmakeFlags+=" PLUGINDIR=$out/$qtPluginPrefix"
-  '';
 
   enableParallelBuilding = true;
 

